src/app/tests/suites/TestDiagnosticLogs-darwin.yaml

```diff
@@ -0,0 +1,515 @@
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        .......................................................
+        ......................................................."
+
+tests:
+    #
+    # Set up the test by adding some destination log files for the target accessory:
+    #  1. End User Support
+    #  2. Network Diagnostics
+    #  3. Crash
+    #
+    # The first thing to do is to delete them if they exist. It could be some
+    # left over from a previous test run.
+    #
+
+    - label: "Delete EndUserSupport logs"
+      cluster: "SystemCommands"
+      command: "DeleteFile"
+      arguments:
+          values:
+              - name: "filePath"
+                value: end_user_support_log_file_path
+
+    - label: "Delete NetworkDiag logs"
+      cluster: "SystemCommands"
+      command: "DeleteFile"
+      arguments:
+          values:
+              - name: "filePath"
+                value: network_diagnostics_log_file_path
+
+    - label: "Stop the accessory"
```

Comment:
Probably better to not use the word "accessory" anywhere in this file....  "stop the server"?  "stop the example app"?  Something else?